src/Microsoft.VisualStudio.Threading.Analyzers.CSharp/VSTHRD110ObserveResultOfAsyncCallsAnalyzer.cs

```diff
@@ -42,6 +40,112 @@ public override void Initialize(AnalysisContext context)
             context.RegisterSyntaxNodeAction(Utils.DebuggableWrapper(this.AnalyzeInvocation), SyntaxKind.InvocationExpression);
         }
 
+        private static bool IsAwaitable(ITypeSymbol returnedSymbol) =>
```

Comment:
The method name suggests it tests whether the given symbol is awaitable, when if fact it only tests whether the symbol is one of a closed set of 4 awaitable things. Also the parameter name could use being more generic, since the fact that it is a "returned symbol" is probably unnecessary, right?

Should we rename the method to `IsStockAwaitable`, perhaps, and rename the parameter to just `symbol`?